content/reference/rest-api/templates/get.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Templates: get'
+weight: 10
+
+menu:
+  main:
+    name: 'get'
+    identifier: 'rest-api-templates-get'
+    parent: 'rest-api-templates'
+    pre: 'Gets a template by ID.'
+---
+
+Gets a template by ID.
```

Comment:
I could imagine that by default only published templates are returned and we offer a parameter to return the unpublished version of a template. This could also apply to the list endpoint.